uptane-standard.md

```diff
@@ -880,23 +880,22 @@ If {{TAP-5}} is supported, a primary ECU SHALL download metadata and images foll
-    2. For each repository listed in this mapping, if top-level metadata has not been refreshed for that repository using the process in {{refresh_toplevel_metadata}} in this iteration of the {{full_verification}} procedure, refresh top-level metadata for that repository using the steps in {{refresh_toplevel_metadata}}.
-    3. For each repository listed in this mapping, perform the procedure in {{resolve_delegations}}, beginning at the top-level Targets metadata in that repository, to find metadata for the image if it exists, and **store** it if it does.
-    4. Find the first set of at least X repositories -- where X is the mapping's threshold, and such that the Director repository is in the set -- for which all of the following is true for the stored image metadata for each repository:
-        1. The non-custom Targets metadata (i.e., length and hashes) of the unencrypted image are the same.
-        2. If listed, the "MUST match" custom Targets metadata (e.g., hardware identifier and release counter) are the same.
-        3. The ECU Identifier is the same across repositories, for all repositories that list it.
-    5. If there is no such set (of at least the mapping's threshold number of repositories meeting the criteria in the prior step), then check the terminating property on this mapping.  If the terminating value for this mapping is true, then abort the search for image metadata for this image, discarding metadata **stored** for this image, as there is no verifiable update for it, and proceed to the next image for which there is **stored** image metadata, if any.  If the terminating value for this mapping is false, continue the search for verified image metadata for this image by continuing to the next mapping, discarding the image metadata for this image from the repositories other than the Director repository.
-7. The image metadata **stored** for each image is now verified.
+2. Refresh the top-level metadata for the Director repository, by following the steps in {{refresh_toplevel_metadata}}. The first repository listed in the repository mapping metadata is always the Director repository.
+3. Iterate through the image metadata listed in the Director repository's Targets metadata file, and **store** the image metadata this ECU is responsible for verifying. A primary ECU is responsible for verifying ALL image metadata; a secondary ECU is responsible for verifying only image metadata that includes its own ECU identifier and hardware ID.
+4. For all images this ECU is responsible for, look for matching image metadata from other repositories until the threshold listed in the repository mapping metadata is met, **or** until a terminating mapping fails. The repository mapping metadata contains a list of mappings that determine which repositories are required to have signed metadata for a particular image. For each piece of image metadata this ECU stored in the previous step, iterate through those mappings as follows:
```

Comment:
This adds some useful background info. 👍

I'd note that because this seems to lay out the end conditions on the iteration (thresholds are met or terminating mapping fails), there might be some confusion about the other ways this iteration can be aborted... but probably not.  (Could use third set of eyes)

Reply:
> What other repositories? Why not just say the Image repository?

Because we may have more than one other repository; clarifying the multi-repo case is what this whole PR is about.